_posts/randy/2022-07-03-all-old-movies-are-garbage.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "All Old Movies Are Garbage (In the End)"
+permalink: /@randy/old-movies-are-garbage
+image: "assets/img/old-movies-are-garbage.png"
```

Comment:
This image hasn't been added from what I can see

Reply:
video was not settled; uploaded now